script.tvmaze.scrobbler/service.py

```diff
@@ -18,10 +18,18 @@
 
 from __future__ import absolute_import, unicode_literals
 
-from libs.kodi_monitor import UpdateMonitor
+import xbmc
+
+from libs.exception_logger import log_exception
+from libs.kodi_monitor import KodiMonitor
 from libs.kodi_service import logger
+from libs.scheduled_tasks import periodic_pull
 
-MONITOR = UpdateMonitor()
-logger.info('Service started')
-MONITOR.waitForAbort()
-logger.info('Service stopped')
+with log_exception():
+    xbmc.sleep(3000)
```

Comment:
Maybe you can also consider changing this one (although the pause time is small)